### PR TITLE
fix: re-add removed code for awarding points on deck and q…

### PR DESCRIPTION
**Pull request by yash431garg**
- Description
In this PR https://github.com/gator-labs/chomp/commit/941bc2835c4320b7fa55f651b9ad6742ab401611#diff-4091e4ffbbcf2f227f3e4bc898c3b28853aa8c67d6e5fd155216ecff80756055L32 mistakenly the code for incrementing points for answer deck and question also got removed. 
- What are the steps to test that this code is working?
You will get points again for answering question and completing a deck
- Screen shots or recordings for UI changes
N/A

### CODE REVIEW

**Review by marvinmarnold — COMMENTED** [review]
Please add some test cases

File: app/actions/fungible-asset.ts [hunk 1]
```diff
@@ -25,6 +25,49 @@ export const getTransactionHistory = async () => {
   return transactionHistory;
 };
 
+interface IncrementFungibleAssetBalanceProps {
```
Comment (neel-ds) [inline app/actions/fungible-asset.ts:28]: Would it be better to move the interface to the types directory?

Comment (yash431garg) [reply app/actions/fungible-asset.ts:28]: Not sure as it is only used here. What you will suggest?

Comment (neel-ds) [reply app/actions/fungible-asset.ts:28]: Make sense if it's not reusable

File: __tests__/actions/answer.test.ts [hunk 2]
```diff
@@ -0,0 +1,147 @@
+import { answerQuestion } from "@/app/actions/answer";
+import { deleteDeck } from "@/app/actions/deck/deck";
+import { getJwtPayload } from "@/app/actions/jwt";
+import prisma from "@/app/services/prisma";
+import { generateUsers } from "@/scripts/utils";
+
+// mock unrealted implementations
```
Comment (neel-ds) [inline __tests__/actions/answer.test.ts:7]: Can you correct the spelling mistake and capitalize it?

File: __tests__/actions/answer.test.ts [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        userId: user[0].id,
+      },
+    });
+
+    const hasAnswerQuestion = res.some(
+      (entry) => entry.type === "AnswerQuestion",
+    );
+    const hasAnswerDeck = res.some((entry) => entry.type === "AnswerDeck");
+
+    expect(hasAnswerQuestion).toBe(true);
```
Comment (neel-ds) [inline __tests__/actions/answer.test.ts:144]: Can you assert points as we have fixed points per action?

Comment (yash431garg) [reply __tests__/actions/answer.test.ts:144]: Sure

**Review by marvinmarnold — COMMENTED** [review]
@yash431garg have you tested in staging since Devin fixed?

**Comment by yash431garg** [comment]
> @yash431garg have you tested in staging since Devin fixed?

No, I have to test.

**Comment by marvinmarnold** [comment]
> No, I have to test.

Cool. Will get this merged to production as soon as it's tested.

**Comment by yash431garg** [comment]
> > No, I have to test.
> 
> Cool. Will get this merged to production as soon as it's tested.

I have tested it ready to be merged.

**Pull request merged**